waiter/resources/web/welcome.html

```diff
@@ -0,0 +1,165 @@
+<!doctype html>
+<html>
+<head>
+  <title>Welcome - Waiter</title>
+  <meta http-equiv="x-ua-compatible" content="IE=edge" />
+  <style type="text/css">
+   html {
+     font-family: 'Lato', 'Helvetica Neue', Arial, Helvetica, sans-serif;
+     color: rgba(0,0,0,.87);
+   }
+
+   .main-content {
+     margin: auto;
+     margin-top: 4em;
+     width: 800px;
+   }
+
+   .main-content h1 {
+     font-size: 1.6rem;
+   }
+
+   .main-content p {
+     font-size: 1.2rem;
+     margin-left: 1em;
+   }
+
+   .divider {
+     /* ui divider */
+     border-top: 1px solid rgba(34,36,38,.15);
+     border-bottom: 1px solid rgba(255,255,255,.1);
+     margin: 1rem 0;
+     line-height: 1;
+     height: 0;
+     font-weight: 700;
+     text-transform: uppercase;
+     letter-spacing: .05em;
+     color: rgba(0,0,0,.85);
+   }
+
+   .additional-info {
+     width: 800px;
+     margin: auto;
+     margin-top: 2em;
+   }
+
+   .additional-info p {
+     margin-left: 1em;
+   }
+
+   .additional-info h3 {
+     margin-top: 2em;
+   }
+
+   .additional-info li {
+     line-height: 2em;
+   }
+
+   .code {
+     font-family: "Lucida Console", Monaco, monospace;
+     line-height: 2em;
+     font-size: 0.9rem;
+   }
+
+   pre.code {
+     margin-left: 1em;
+     margin-right: 1em;
+     white-space: pre-wrap;
+     font-size: 0.8rem;
+   }
+
+   .additional-info td {
+     padding-right: 1em;
+   }
+
+   .additional-info td.field {
+     font-weight: bold;
+     text-align: right;
+     vertical-align: baseline;
+     width: 70px;
+   }
+
+   .additional-info td.code {
+     vertical-align: baseline;
+   }
+
+   .additional-info table {
+     margin-left: 1em;
+     border-collapse: collapse;
+   }
+    </style>
+</head>
+<body>
+
+<div class="main-content">
+  <h1>
+    Welcome to Waiter
+  </h1>
+  <p>
+    Waiter is a platform for running and managing web services and applications.
+  </p>
+  <p>
```

Comment:
Do we need to wrap this with `when support-info`?

Reply:
Done